sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/OperationStatusResult.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> The current status of an async operation. </summary>
+    public partial class OperationStatusResult
```

Comment:
Can we rename it to `WorkloadsOperationStatusResult`? I saw it was uesd in different `delete`, `start` or `stop` method, maybe add a RP prefix is better.